mapswipe_workers/mapswipe_workers/utils/api_calls.py

```diff
@@ -13,7 +18,74 @@ def geojsonToFeatureCollection(geojson):
     return geojson
 
 
-def ohsome(request: dict, area: str, properties=None):
+def query_osm(changeset_id: int):
+    """Get data from changesetId."""
+    url = OSM_API_LINK + f"changeset/{changeset_id}.json"
+    valid_response = False
+    times_timed_out = 0
+    while not valid_response:
```

Comment:
looks interesting. So you want to try again after a timeout, correct?

maybe check this blog post how to achieve it a bit nicer: https://www.peterbe.com/plog/best-practice-with-retries-with-requests

Reply:
done